opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -114,6 +114,34 @@ def test_start_span_explicit(self):
             self.assertIs(tracer.get_current_span(), root)
             self.assertIsNotNone(child.end_time)
 
+    def test_span_members(self):
+        context = contextvars.ContextVar('test_span_members')
+        tracer = trace.Tracer(context)
+
+        other_neighbor = trace_api.SpanContext(
+            trace_id=0x0000000000000000000000000000cafe,
```

Comment:
thoughts on using the sdk's trace.get_trace_id() and trace.get_span_id()?

Reply:
I'll use `trace.generate_trace_id()` and `trace.generate_span_id()`.